cores/arduino/Uart.cpp

```diff
@@ -91,8 +91,11 @@ void Uart::flush()
 void Uart::IrqHandler()
 {
   if (sercom->availableDataUART()) {
-    rxBuffer.store_char(sercom->readDataUART());
-
+    sercom->isFrameErrorUART();
```

Comment:
This line doesn't do anything useful, just read the error and ignore.

Should we also clear the frame error?